writing-guide/readme.md

```diff
@@ -0,0 +1,78 @@
+> **Goal of this guide:** To help team members write in a clear manner
+
+# How to write well
+
+## Explain the content of the article
+
+We are trained to filter information very efficiently and are always thinking about how to best spend our valuable time. Consequently, we want to know beforehand that the time we spend on an activity will be rewarded. You should, therefore, grab the attention of the reader very early on. Only once interest has been piqued can you be sure that the reader will want to devote attention to the rest of the article, which will lead to a gratifying reading experience, as promised.
+
+You make a promise to the reader in the beginning of the article; be sure to fulfill that promise and offer a nice ride towards it!
+
+## Guide the reader through your thought process
+
+You have a lot of knowledge that most readers don't 💾. While explaining something, then, you should always guide the reader through your whole thought process. There are no shortcuts. Use supporting facts, descriptions, and examples to clarify your ideas. Explain everything the reader needs to know but nothing more–too much information is off-putting. The **article should be as concise as possible**. This is difficult, so take your time! 🕗
+
```

Comment:
"more–too much information is off-putting"

more-too reads as 1 word now. Add a space?